tests/test_error_cmd.py

```diff
@@ -1,44 +1,43 @@
-import pytest
+from ragger.backend import RaisePolicy
 
 from boilerplate_client.exception import errors, DeviceException
 
 
-def test_bad_cla(cmd):
-    sw, _ = cmd.transport.exchange(cla=0xa0,  # 0xa0 instead of 0xe0
-                                   ins=0x03,
-                                   p1=0x00,
-                                   p2=0x00,
-                                   cdata=b"")
+def test_bad_cla(backend, firmware):
+    backend.raise_policy = RaisePolicy.RAISE_NOTHING
+    rapdu = backend.exchange(cla=0xa0,  # 0xa0 instead of 0xe0
+                             ins=0x03,
+                             p1=0x00,
+                             p2=0x00,
+                             data=b"")
+    assert DeviceException.exc[rapdu.status] == errors.ClaNotSupportedError
```

Comment:
Why not using simply 
`assert rapdu.status == errors.ClaNotSupportedError`

And in my opinion, we can have a simple error enum instead of a class per error. What do you think about it?

Reply:
`errors.ClaNotSupportedError` is a class, not an int, I need to convert `rapdu.status` first before comparing

> And in my opinion, we can have a simple error enum instead of a class per error. What do you think about it?

Absolutely agree but out of the scope of my work on NeoN3

Reply:
> Absolutely agree but out of the scope of my work on NeoN3

I'm not sure if I agree with this as I doubt anyone will change it later.